src/Modules/UngradedModules.jl

```diff
@@ -161,6 +190,9 @@ Return  `true` if `F` and `G` are isomorphic, `false` otherwise.
 Here, `F` and `G` are isomorphic iff their base rings and ranks are equal.
 """
 function is_isomorphic(F::FreeMod, G::FreeMod)
+  @assert rank(F) == rank(G)
```

Comment:
shouldn't this just return false? Why is it an error?

Reply:
Yes, thanks for spotting it.